.yarn/versions/a1a586ad.yml

```diff
@@ -1,2 +0,0 @@
-releases:
-  "@hashicorp/pds-ember": prerelease
```

Comment:
RELEASE THE 🐙

Reply:
FYI: These yml files will need to be updated with every PR, from now on.